src/USER-REAXC/reaxc_forces.cpp

```diff
@@ -161,9 +163,10 @@ void Validate_Lists( reax_system *system, storage * /*workspace*/, reax_list **l
         else comp = hbonds->num_intrs;
 
         if (End_Index(Hindex, hbonds) > comp) {
-          fprintf(stderr,"step%d-hbondchk failed: H=%d end(H)=%d str(H+1)=%d\n",
+          char errmsg[256];
+          snprintf(errmsg, 256, "step%d-hbondchk failed: H=%d end(H)=%d str(H+1)=%d\n",
                   step, Hindex, End_Index(Hindex,hbonds), comp );
-          MPI_Abort( comm, INSUFFICIENT_MEMORY );
+          system->error_ptr->all(FLERR, errmsg);
```

Comment:
same as in USER-OMP: does this error really get always triggered by all MPI ranks?